### PR TITLE
[change] Adjust AB relative imports

### PR DESCRIPTION
**Pull request by dgdekoning**
Related to https://github.com/cauldron/activity-browser/issues/9, this MR changes the AB related imports to only use relative imports if they are within the same folder.

Additionally, I've moved the `ABTab` class up into a `base.py` file and combined all 'panel' classes into a single `panels.py` file.

Due to the likelihood of `bwutils` changing in the future I haven't made any import changes there.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation, please follow the [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] Update tests.
- [ ] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `feature`, `ui`, `change`, `documentation`, `breaking`, `ci`
      as they show up in the changelog
- [x] Link this PR to related issues.

### CODE REVIEW

File: activity_browser/controllers/database.py [hunk 1]
```diff
@@ -2,26 +2,25 @@
 import logging
 
 import brightway2 as bw
-from bw2data.backends.peewee import sqlite3_lci_db
+from bw2data.backends.proxies import sqlite3_lci_db
```
Comment (cmutel) [inline activity_browser/controllers/database.py:5]: from bw2data.backends import sqlite3_lci_db

File: activity_browser/application.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 import logging
 
-from .controllers import controllers
-from .layouts import MainWindow
-from .logger import ABHandler
+from activity_browser.controllers import CONTROLLERS
+from activity_browser.layouts.main import MainWindow
+from activity_browser.logger import ABHandler
```
Comment (cmutel) [inline activity_browser/application.py:6]: You clearly explain the what in the PR comment, but not the why... what does changing from relative to absolute imports do here?

Comment (dgdekoning) [reply activity_browser/application.py:6]: The why is mostly clarity for other people/developers, absolute imports make it easier to follow where methods/classes come from. And with further refactoring it should be simpler to move code around without having to continuously adjust the import statements.

File: activity_browser/controllers/database.py (hunk 1 above)
Comment (dgdekoning) [reply activity_browser/controllers/database.py]: Corrected!

**Comment by cmutel** [comment]
Out of date, easier to start over given the changes

**Pull request closed without merging**